docs/index.rst

```diff
@@ -174,6 +174,7 @@ Using Jdaviz
   imviz/index.rst
   specviz/index.rst
   cubeviz/index.rst
+  specviz2d/index.rst
```

Comment:
Need to add specviz2d stuff to API docs too?

Also add new Specviz2D section to change log template in the release instructions?

On GitHub itself, should I create a new Specviz2D label and add corresponding Issue and Bug templates?

Reply:
Since specviz2d is currently used by MAST (which links to a non-existent RTD page), I think we should... and we can always merge with mosviz later if we decide to go that route.  Full docs are out-of-scope for this PR, I think/hope, but I just put the bare minimum so that all the plugin links point somewhere appropriate.

I added a specviz2d label in GitHub already (for this PR) and added a custom section in the change log (but not the template).  The [helper](https://jdaviz.readthedocs.io/en/latest/api/jdaviz.configs.specviz2d.helper.Specviz2d.html#jdaviz.configs.specviz2d.helper.Specviz2d) was already in the docs, but I'll add this plugin as well, thanks for catching that!

Reply:
The book-keeping stuff (templates, release instructions) can be done after this is merged. Thanks!

Reply:
I'll let @rosteen decide if anything in JIRA needs updating to include specviz2d...